Modules/ITS/src/ITSDecodingErrorCheck.cxx

```diff
@@ -29,15 +30,41 @@ namespace o2::quality_control_modules::its
 
 Quality ITSDecodingErrorCheck::check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap)
 {
+  std::vector<int> vDecErrorLimits = convertToArray<int>(o2::quality_control_modules::common::getFromConfig<string>(mCustomParameters, "DecLinkErrorLimits", ""));
+  if (vDecErrorLimits.size() != o2::itsmft::GBTLinkDecodingStat::NErrorsDefined) {
+    LOG(error) << "Incorrect vector with DecodingError limits, check .json" << ENDM;
+    doFlatCheck = true;
+  }
+
   Quality result = Quality::Null;
 
   for (auto& [moName, mo] : *moMap) {
     (void)moName;
     if ((mo->getName() == "General/LinkErrorPlots") || (mo->getName() == "General/ChipErrorPlots")) {
       result = Quality::Good;
       auto* h = dynamic_cast<TH1D*>(mo->getObject());
-      if (h->GetMaximum() > 200) {
-        result.set(Quality::Bad);
+
+      if (doFlatCheck || mo->getName() == "General/ChipErrorPlots") {
```

Comment:
shouldn't you put `if( doFlatCheck && mo->getName() == "General/ChipErrorPlots")`?

Reply:
Basically, I wrote it this way before, because I expect to always apply flat threshold for ChipErrorPlots, since we do not use it for shifters. 

I have fixed the code, so it should be more clear to read.